src/app/controllers/MachineController.ts

```diff
@@ -122,18 +126,19 @@ export default class MachineController {
 			})
 		}
 
-		chartData.push(this.addBytesConfig())
-
-		return chartData
+		return {
+			Data: chartData,
+			Config: this.addBytesConfig(),
+		} as MachineChartData
 	}
 
-	public doCPUCharts(current: ProcessedStats): any[] {
+	public doCPUCharts(current: ProcessedStats): MachineChartData {
 		const chartData = []
 
-		if (!current) return chartData
-		if (!current.system) return ['system_missing']
+		if (!current) return { Data: chartData } as MachineChartData
+		if (!current.system) return { Data: chartData, Error: 'system_missing' } as MachineChartData
```

Comment:
Why does this return Data? Compare it with, for example, `doMemorySystemCharts`.
Before the changes it did not return data either.

Reply:
fixed